rskj-core/src/main/java/co/rsk/rpc/modules/eth/subscribe/EthSubscribeLogsParams.java

```diff
@@ -41,6 +41,7 @@ public EthSubscribeLogsParams() {
     }
 
     @JsonCreator
+    // TODO: modify?
```

Comment:
@fedejinich should I modify this class as well?
I assume clients can subscribe to events and receive logs somehow, and this is what it's for (or maybe it's deprecated, I really can't tell).